site/en/docs/android/custom-tabs/howto-custom-tab-request-headers/index.md

```diff
@@ -1,6 +1,7 @@
 ---
 layout: "layouts/doc-post.njk"
-title: How to add extra HTTP Request Headers to Custom Tab Intents
+title: How to add extra HTTP Request Headers 
```

Comment:
The norms for titles in Custom Tabs was established long before you came on the scene. For future reference, think about what happens if you have a page full of titles that begin with the same words. It adds friction for someone scanning for a specific topic.

Reply:
Thanks, missed renaming this.